### DISCUSSION TITLE
IDE Charging money on every context submit that times-out. Feeling robbed...

### DISCUSSION THREAD

**Discussion by creslinux** [zed-industries/zed#19465] [Feature Requests]
I typically get timeouts a few times in a row in large context windows, i thought this fine and resubmit with a `?` i get the answer after a few retries. 
Today the IDE asked I subscribe as id used my free-allowance. Great service - why not. 
Put my card details in 

Resent my context a 1, 2, 3 times - I can see my account getting charged in real time for every non-response / timeout , costing over 0.2$ for each request and nothing returned. 

ZED should not be charging cash when it does not respond for what ever reason. 

How to resolve this? 
Thanks

![Screenshot From 2024-10-19 09-53-45](https://github.com/user-attachments/assets/c20ec997-5f10-4a36-a582-235d13d05ca0)

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
